volmaster/daemon.go

```diff
@@ -71,25 +71,33 @@ func (d *DaemonConfig) Daemon(listen string) {
 	r := mux.NewRouter()
 
 	postRouter := map[string]func(http.ResponseWriter, *http.Request){
-		"/request":      d.handleRequest,
-		"/create":       d.handleCreate,
-		"/copy":         d.handleCopy,
-		"/mount":        d.handleMount,
-		"/mount-report": d.handleMountReport,
-		"/unmount":      d.handleUnmount,
-		"/remove":       d.handleRemove,
+		"/global":                           d.handleGlobalUpload,
+		"/volumes/create":                   d.handleCreate,
+		"/volumes/copy":                     d.handleCopy,
+		"/volumes/remove":                   d.handleRemove,
+		"/volumes/removeforce":              d.handleRemoveForce,
+		"/volumes/request":                  d.handleRequest,
+		"/mount":                            d.handleMount,
+		"/mount-report":                     d.handleMountReport,
+		"/policies/{policy}":                d.handlePolicyUpload,
+		"/policies/delete/{policy}":         d.handlePolicyDelete,
+		"/runtime/{policy}/{volume}":        d.handleRuntimeUpload,
```

Comment:
This was also requested to not be done because it's related to some 
critical etcd operations. The CRUD stuff is what I asked him to complete 
in this patch.

On 24 May 2016, at 9:28, Madhav Puri wrote:

>> -		"/copy":         d.handleCopy,
>> -		"/mount":        d.handleMount,
>> -		"/mount-report": d.handleMountReport,
>> -		"/unmount":      d.handleUnmount,
>> -		"/remove":       d.handleRemove,
>> +		"/global":                           d.handleGlobalUpload,
>> +		"/volumes/create":                   d.handleCreate,
>> +		"/volumes/copy":                     d.handleCopy,
>> +		"/volumes/remove":                   d.handleRemove,
>> +		"/volumes/removeforce":              d.handleRemoveForce,
>> +		"/volumes/request":                  d.handleRequest,
>> +		"/mount":                            d.handleMount,
>> +		"/mount-report":                     d.handleMountReport,
>> +		"/policies/{policy}":                d.handlePolicyUpload,
>> +		"/policies/delete/{policy}":         d.handlePolicyDelete,
>> +		"/runtime/{policy}/{volume}":        d.handleRuntimeUpload,
>
> similar to mount/unmount may be `/runtime/{policy}/{volume}` should be 
> `/volumes/runtime/{policy}/{volume}` ?
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/contiv/volplugin/pull/255/files/12004ac3ec0c06ee7937e32845ee33e7c45a120f#r64426382